### DISCUSSION TITLE
Failed to export bib files due to translators?

### DISCUSSION THREAD

**Discussion by lyiqian** [retorquere/zotero-better-bibtex#2832] [Q&A]
When I tried exporting the bib file (Better BibTex) for two items in my zotero library, I got a popup saying an error occured. Below attached is the debug logging. I am resetting the translators, meanwhile any one could give some insight? Thank you!

***
(3)(+0000007): Getting contents of /home/yiqian/yichian-gdrive/zotero-references/translators/Better BibTeX.js

(4)(+0000027): Translate: Parsing code for Better BibTeX (ca65189f-8815-4afe-8c8b-8c7c15f0edca, 2024-02-13)

(1)(+0000003): SyntaxError: expected expression, got keyword 'try' eval/<@chrome://zotero/content/xpcom/translation/translate_firefox.js:425:7 eval@chrome://zotero/content/xpcom/translation/translate_firefox.js:424:6 Zotero.Translate.Base.prototype._loadTranslator</parse<@chrome://zotero/content/xpcom/translate/src/translation/translate.js:1782:5

(2)(+0000001): Translate: Translation using Better BibTeX failed: SyntaxError: expected expression, got keyword 'try' eval/<@chrome://zotero/content/xpcom/translation/translate_firefox.js:425:7 eval@chrome://zotero/content/xpcom/translation/translate_firefox.js:424:6 Zotero.Translate.Base.prototype._loadTranslator</parse<@chrome://zotero/content/xpcom/translate/src/translation/translate.js:1782:5 url => /home/yiqian/Downloads/Exported Items.bib downloadAssociatedFiles => true automaticSnapshots => true

(5)(+0000001): Translate: Running handler 0 for error

(5)(+0000002): Translate: Running handler 0 for done

(3)(+0000000): Running unlock callbacks

(3)(+0000001): Alert: An error occurred while trying to export the selected file.

**Comment by retorquere** [comment]
You need to open an issue for this (unless you are on the latest beta, then you can subscribe to #2829)
